src/ListViewSwipe.ts

```diff
@@ -247,6 +259,18 @@ class ListViewSwipe extends WidgetBase {
         }
     }
 
+    private callNanoflow(direction: Direction, context: mendix.lib.MxContext) {
+        if (this.onSwipeAction[direction] === "callNanoflow" && this.onSwipeNanoflow[direction].nanoflow) {
+            window.mx.data.callNanoflow({
+                context,
+                error: error => window.mx.ui.error(`An error occurred while executing action 
+                    ${this.onSwipeNanoflow[direction]}: ${error.message}`, true),
+                nanoflow: this.onSwipeNanoflow[direction],
+                origin: this.mxform

```

Comment:
Re-organise properties based on mendix client, i.e 
    `params: {
        actionname: ""
    },
    origin: this.mxform,
    callback: (),
    error: (error)`

Reply:
Done